tests/Unit/Evolution/Systems/Cce/AnalyticSolutions/GaugeWave.py

```diff
@@ -0,0 +1,93 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import numpy as np
```

Comment:
Only a suggestion, entirely up to you: In math-heavy Python files it can be useful to `from numpy import pi, sin, cos, exp` so you don't have to sprinkle those `np.` everywhere.

Reply:
but namespacing.... I find this is an interesting suggestion since it goes against the advice given in the C++ community :) Is the advice of "don't do `using namespace...`" not a thing in Python?

Reply:
My understanding from a mild exposure to python is that when you do `from blah import foo` stuff, it's actually regarded as 'good enough' for the namespace philosophy, because it is always explicitly in the file what you are using, and a lot of editors (thankfully emacs too...) have good packages for working out the set of objects and functions that are brought in from such calls.
My understanding is that it is considered less accepted practice to do the (nonetheless fairly common) `from numpy import *` because it's impossible to work out the full set of garbage that brings into the local scope.

Reply:
ah yea, that makes sense. doing `using std::max` is a lot less bad than bringing in the whole namespace. Thanks for the explanation! :)

Reply:
Exactly, in fact most (all?) linters will shout loudly at you if you try to `from x import *`